### PR TITLE
Platformer Update

### PR DESCRIPTION
**Pull request by MTM828**
...

### CODE REVIEW

**Comment by sparshg** [comment]
wait how did you open PR for just the latest commit instead of whole branch???????? Did you run `git cherry-pick` command? (If you see the file changes, there's only a single line change)

**Comment by sparshg** [comment]
okay finally, lol I had to rebase this branch with main

File: src/platformer/platformer.py [hunk 1]
```diff
@@ -3,41 +3,53 @@
 Github repo can be found here:
 https://github.com/sparshg/pycollab
 """
+
+# Import statements
+import pygame as pg
 from os import environ
 
 # Hide pygame Hello prompt
 environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
```
Comment (sparshg) [inline src/platformer/platformer.py:12]:
```suggestion

from os import environ

# Hide pygame Hello prompt
environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
import pygame as pg
```
pygame should be imported after hiding the prompt or it won't hide that

**Review by sparshg — APPROVED** [review]

File: src/platformer/platformer.py (hunk 1 above)
Comment (MTM828) [reply src/platformer/platformer.py]: Ah, got it, thanks.

Comment (sparshg) [reply src/platformer/platformer.py]: soo, merge?

Comment (MTM828) [reply src/platformer/platformer.py]: OK.
But can't we perhaps continue coding this when not so busy?

Comment (sparshg) [reply src/platformer/platformer.py]: ok, you can message me when you get free from exams so we can continue

Comment (MTM828) [reply src/platformer/platformer.py]: Wow, cool site! :D

**Pull request merged**